src/WordPress/Hooks.php

```diff
@@ -82,47 +79,17 @@ public function initProxy()
 
     public function activate()
     {
-        $this->checkVersionCompatibility();
-        $this->checkDependenciesExist();
```

Comment:
Why did we remove the functions? Functions provide self-documentation and make the code more readable.